composer.json

```diff
@@ -16,7 +16,7 @@
         "padraic/humbug_get_contents": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~6.0"
```

Comment:
You're right, I need to allow for a lesser version to resolve deps for composer on Travis for PHP 5.6.